From d989a82e4fd2c22182bb1d98b7873e540aa73b78 Mon Sep 17 00:00:00 2001
From: Angus Thomas <athomas@redhat.com>
Date: Thu, 27 Feb 2014 16:49:05 +0000
Subject: [PATCH] Remove runtime dependency on python pbr

---
 ironicclient/__init__.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ironicclient/__init__.py b/ironicclient/__init__.py
index 73028d4..0b5a2ef 100644
--- a/ironicclient/__init__.py
+++ b/ironicclient/__init__.py
@@ -15,13 +15,12 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import pbr.version
 
 from ironicclient import client
 from ironicclient import exc as exceptions
 
 
-__version__ = pbr.version.VersionInfo('python-ironicclient').version_string()
+__version__ = "REDHATIRONICCLIENTVERSION"
 
 __all__ = [
         'client',
-- 
1.9.3

