From 628694a030ea37bb80b2e31dd2d0494c3f0c1989 Mon Sep 17 00:00:00 2001
From: Angus Thomas <athomas@redhat.com>
Date: Thu, 27 Feb 2014 16:49:05 +0000
Subject: [PATCH] Remove runtime dependency on python pbr

---
 ironicclient/__init__.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/ironicclient/__init__.py b/ironicclient/__init__.py
index 4aa915f..2c4fe3d 100644
--- a/ironicclient/__init__.py
+++ b/ironicclient/__init__.py
@@ -13,7 +13,4 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import pbr.version
-
-
-__version__ = pbr.version.VersionInfo('python-ironicclient').version_string()
+__version__ = "REDHATIRONICCLIENTVERSION"
-- 
1.8.5.3

