From 03de324ed30440fa5eeaf27086982293c0e630c5 Mon Sep 17 00:00:00 2001
From: Angus Thomas <athomas@redhat.com>
Date: Thu, 16 Oct 2014 18:07:36 +0100
Subject: [PATCH] Remove pbr dep

---
 ironicclient/__init__.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ironicclient/__init__.py b/ironicclient/__init__.py
index 73028d4..0b5a2ef 100644
--- a/ironicclient/__init__.py
+++ b/ironicclient/__init__.py
@@ -15,13 +15,12 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import pbr.version
 
 from ironicclient import client
 from ironicclient import exc as exceptions
 
 
-__version__ = pbr.version.VersionInfo('python-ironicclient').version_string()
+__version__ = "REDHATIRONICCLIENTVERSION"
 
 __all__ = [
         'client',
-- 
1.9.3

